### PR TITLE
Add a change-log entry for the simplex efforts

### PR DESCRIPTION
**Pull request by peterrum**
Depends on the simplex tutorial (PR #11581) and mixed-mesh tutorial (not started yet).

### CODE REVIEW

**Review by bangerth — APPROVED** [review]

**Comment by bangerth** [comment]
The "indent" check fails with a doxygen error. Other than that ok to merge.

**Comment by peterrum** [comment]
> The "indent" check fails with a doxygen error. Other than that ok to merge.

That is because the referenced tutorials are missing ;) I would suggest to wait!

**Comment by peterrum** [comment]
Ready from my side.

File: doc/news/changes/major/20210304various [hunk 1]
```diff
@@ -0,0 +1,8 @@
+New: An experimental support for simplex meshes (i.e., meshes consisting
+of triangle or tetrahedron cells) and mixed meshes (i.e., meshes consisting
+of lines, triangles, quadrilateral, tetrahedron, pyramid, wedge, and/or 
+hexahedron cells) has been added. The new features are presented in 
+variants of multiple tutorials in test test folder tests/simplex and in the
```
Comment (bangerth) [inline doc/news/changes/major/20210304various:5]:
```suggestion
variants of multiple tutorials in test folder `tests/simplex` and in the 
```

Comment (bangerth) [reply doc/news/changes/major/20210304various:5]: Just drop the part with the tutorials.

File: doc/news/changes/major/20210304various [hunk 2]
```diff
@@ -0,0 +1,8 @@
+New: An experimental support for simplex meshes (i.e., meshes consisting
```
Comment (bangerth) [inline doc/news/changes/major/20210304various:1]:
```suggestion
New: Experimental support for simplex meshes (i.e., meshes consisting
```

File: doc/news/changes/major/20210304various [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+of triangle or tetrahedron cells) and mixed meshes (i.e., meshes consisting
+of lines, triangles, quadrilateral, tetrahedron, pyramid, wedge, and/or 
+hexahedron cells) has been added. The new features are presented in 
+variants of multiple tutorials in test test folder tests/simplex and in the 
+new ``Simplex'' module page.
```
Comment (bangerth) [inline doc/news/changes/major/20210304various:6]: Doxygen doesn't understand latex markup for quotes.
```suggestion
new "Simplex" module page.
```

**Comment by peterrum** [comment]
@bangerth Done!

**Review by bangerth — APPROVED** [review]

**Pull request merged**